releasenotes/notes/bind_replace-2d64290b8f49ee9e.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Bind ``apm_config.replace_tags`` parameter to ``DD_APM_REPLACE_TAGS`` environment variable.
```

Comment:
@zippolyte we have to prefix all APM-related change log entries with `APM :`, can you please follow up with this change?

Reply:
Done: https://github.com/DataDog/datadog-agent/pull/4017